public/docs/ts/latest/glossary.jade

```diff
@@ -766,12 +747,10 @@ a#jit
     Angular 2 itself is written in TypeScript.
```

Comment:
This is no longer true for dart-lang/angulardart. We need to update this definition (at least the Dart version, preferably the JS/TS version as well).